libs/sdk-ui-ext/src/internal/components/configurationPanels/LineChartBasedConfigurationPanel.tsx

```diff
@@ -83,12 +86,8 @@ export default class LineChartBasedConfigurationPanel extends BaseChartConfigura
 
                         <ContinuousLineControl
                             properties={properties}
-                            checked={continuousLineSelectable ? continuousLineEnabled : false}
-                            disabled={
-                                controlsDisabled ||
-                                !continuousLineSelectable ||
-                                isContinuousLineControlDisabled
-                            }
+                            checked={shouldContinuousLineControlDisabled ? false : continuousLineEnabled}
```

Comment:
You could simplify this condition as `!shouldContinuousLineControlDisabled && continuousLineEnabled`. 

Ideally, you should not use ternary condition when one of the branches is boolean primitive (true/false) as these can be always simplified (it takes a lots of thinking for me every time however to write it correctly :D)

Reply:
The checked value should depend on the `continuousLineEnabled` value and it will un-check when the control is disabled. With this code `!shouldContinuousLineControlDisabled && continuousLineEnabled`, the checkbox is always checked when the `shouldContinuousLineControlDisabled` is false. So, I made a small change to make it more clear. Thanks!